lib/src/rules/avoid_relative_lib_imports.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
```

Comment:
I'm guessing that you updated the wrong copyright notice. This one should be reverted.

Reply:
updated